libs/ballot-interpreter/src/hmpb-rust/qr_code/detect.rs

```diff
@@ -235,4 +238,22 @@ mod test {
         assert_eq!(qr_code.bounds(), Rect::new(97, 2016, 108, 107));
         assert_eq!(qr_code.orientation(), Orientation::Portrait);
     }
+
+    #[test]
+    fn test_detect_qr_code_in_skewed_image() {
+        let fixture_path =
+            PathBuf::from(env!("CARGO_MANIFEST_DIR")).join("test/fixtures");
+        let scan_side_a_path = fixture_path.join("scan-skewed-side-a.jpeg");
+        let scan_side_b_path = fixture_path.join("scan-skewed-side-b.jpeg");
+        detect(
+            &image::open(scan_side_a_path).unwrap().into_luma8(),
+            &ImageDebugWriter::disabled(),
+        )
+        .expect("side A QR code not detected");
```

Comment:
When I first read this, I really thought it was an assertion that the expected result should be "side A QR code not detected." I guess this is a pitfall of the `expect` nomenclature

Reply:
I changed it to be clearer since it looks like we mostly use it the other way.